Gaijinizer patch/Map402.txt

```diff
@@ -2,512 +2,624 @@
 
 \\C[2]魔力の感覚を取り戻して、魔法で彼女を撃退すること…！
-> CONTEXT: Map402/events/1/pages/0/283/Dialogue < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/283/Dialogue
+In which case, I've only got one option left...
 
+\\C[2]I need to regain my sense of magic and use spells to
+drive her away!#Ron
 > END STRING
 
 > BEGIN STRING
 いまの彼女は、僕が無力だと思って油断している
 
 \\C[2]ダメージは与えなくていい……吹き飛ばして
 彼女に冷静に話を聞いてもらう状況をつくるんだ…！
-> CONTEXT: Map402/events/1/pages/0/287/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/287/Dialogue
+She'll think that I don't have any magic at my disposal.
+\\C[2]I don't need to deal any damage to her, I just need to
+blow her around enough that she'll start listening to
+reason.#Ron
 > END STRING
 
 > BEGIN STRING
 ああ……あぁぁ…………
-> CONTEXT: Map402/events/1/pages/0/306/Dialogue/enemy27 < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/306/Dialogue/enemy27
+Aah... Aaaah...#Ron
 > END STRING
 
 > BEGIN STRING
 マイは、攻撃されたことが予想外だったのか
 それとも、拒絶されたことがショックだったのか…
 
 膝から崩れ落ち、うつむいたまま動かなくなってしまった…
-> CONTEXT: Map402/events/1/pages/0/311/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/311/Dialogue
+Either from the surprise of my attack, or from the shock
+of being refused, Mai falls to her knees and hangs her
+head.#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]くん………
 私のこと………キライ……？
 
 こんな………ずるくて………強引な………
-> CONTEXT: Map402/events/1/pages/0/321/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/321/Dialogue/enemy27
+\\N[1]-kun...
+Do you... hate me...?
 
+Was it... because we were so forceful...?#Ron
 > END STRING
 
 > BEGIN STRING
 ハッキリ言ってほしい………
 
 そうすれば、私も………諦められる……
-> CONTEXT: Map402/events/1/pages/0/326/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/326/Dialogue/enemy27
+I want you to give it to me straight...
 
+That way I'll be able to give up on you...#Ron
 > END STRING
 
 > BEGIN STRING
 キライ……なんかじゃないよ。
 
 君が僕に向けているのが、『悪意』じゃないって
 僕もちゃんと分かってるから。
-> CONTEXT: Map402/events/1/pages/0/332/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map402/events/1/pages/0/332/Dialogue/kazuya
+I don't hate you or anything...
 
+I know full well that you're not doing this out
+of any ill-will towards me...#Ron
 > END STRING
 
 > BEGIN STRING
 君たち悪魔は、人間を意のままに出来る力を持ってる。
 だから、拒否されていることに慣れていない。
 
 今の君は、心の整理がついていないだけなんだよ。
-> CONTEXT: Map402/events/1/pages/0/337/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/337/Dialogue/kazuya
+You demons have the power to make humans do as
+you please, so you're not used to being told no.
+You just haven't had time to sort out the
+feelings in your heart yet.#Ron
 > END STRING
 
 > BEGIN STRING
 僕だって、これ以上きみを傷付けたくはない…
 本当は、攻撃だってしたくない……
 
 だから、僕のことを思うなら、このまま行かせてくれ……
-> CONTEXT: Map402/events/1/pages/0/342/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map402/events/1/pages/0/342/Dialogue/kazuya
+I don't want to hurt you any more than I already
+have. The truth is I don't want to attack you at
+all. So if you really care for me, please let me
+go on ahead.#Ron
 > END STRING
```

Comment:
Would "hurt" be a better word then "attack" here?

Reply:
Strictly speaking, no. But it probably flows a lot better.